DropboxSync.BLL/Configurations/UploadConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DropboxSync.BLL.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DropboxSync.BLL.Configurations
+{
+    public class UploadConfiguration : IEntityTypeConfiguration<UploadEntity>
```

Comment:
maybe we want to save also metadata about the file, such as the filename, size, content type,etc

Reply:
The path containing the file name is it still worth it ?

Because we have the path to a file, data can always be retrieved, I think adding these data would not be relevant

Reply:
my comment below >